modules/Oculus_module/src/OculusModule.cpp

```diff
@@ -12,13 +12,17 @@
 #include <yarp/os/LogStream.h>
 #include <yarp/os/Property.h>
 #include <yarp/os/Stamp.h>
+#include <yarp/dev/FrameGrabberInterfaces.h>
```

Comment:
Nope. I am just adding the interface.  That file should be always installed by ``yarp``. The respective device instead is the ``dragonfly2``, that should be enabled in order to have the cameras working. 
In our specific case, it is enabled with https://github.com/robotology/robotology-superbuild/blob/master/cmake/BuildICUB.cmake#L53
and then it implements the ``FrameGrabberInterface`` that I need  https://github.com/robotology/icub-main/blob/5d491cf71ac08dc0e7774c81a5dd0f12035af7c6/src/libraries/icubmod/dragonfly2/common/DragonflyDeviceDriver2.h#L291

So, bottom line is that as long as the cameras on the robot are working (and are dragonfly), that should work.

Reply:
OK, I see, thanks for explanation.
So, just to make the path correctly, we have an option `ROBOTOLOGY_ENABLE_ICUB_HEAD ` in superbuild. If this is `on`, then the icub module also installs the device as you pointed out in https://github.com/robotology/robotology-superbuild/blob/master/cmake/BuildICUB.cmake#L53 and we have this option only on ubuntu machine of the icub. Then my question is, what if we are not running the oculus module code without iCub ? What will be the output of `resetCamera` function, considering opening the device and the result of `grabberInterface->setResetDC1394()`?

Reply:
Not sure I understood your concern
In any case:
- the module compiles even if the option ``ROBOTOLOGY_ENABLE_ICUB_HEAD`` is set to ``OFF``. In fact, I tested it also by running the ``OculusRetargetingModule`` on my PC.
- If you are not connected to any robot, the configuration will fail before because of the other retargeting modules.
- If you are connected to a robot, and for some reason you don't want to use the eyes, you can simply set the option ``resetCameras`` to 0. Indeed, by default ``resetCameras`` is false, and I added this parameter only to the ``iCubGenova04`` configuration files. 
- Any problems related to the opening of the camera driver would occur only if  ``resetCameras`` is non-zero.

Reply:
OK, that sounds good. So we cover different situations without causing any problem. Thanks.